src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/TestRunner.cs

```diff
@@ -242,7 +242,7 @@ public TestRunFullResults RunTests()
                 {
                     var engineName = runner.GetName();
 
-                    var (result, msg) = runner.RunAsync(testCase).Result;
```

Comment:
>Result [](http://example.com/codeflow?start=66&length=6)

double check?

Reply:
Intended, runasync isn't actually async (I'll fix the name)